exampleSite/config.toml

```diff
@@ -50,6 +50,11 @@ disqusShortname = "yourdiscussshortname"
     # Custom JS
     custom_js = []
 
+# If you want to use fathom(https://usefathom.com) for analytics, add this section
+[params.fathomAnalytics]
+    siteID = "ABCDE"
+    rootURL = "analytics.example.com"
```

Comment:
I just don't get what is `rootURL` for. It should points to the root of the site or is a link to the analytics panel?

Reply:
It points to the analytics panel, fathom provides a paid service but can be self hosted as well. So, to keep it open-ended, this is being taken as a param

Reply:
@luizdepra Should I reword the variable to something more understandable ?

Reply:
Yup. Maybe serverURL or something like that.
If there is a hosted option for FathomAnalytics and its URLs will not change by user, you can provide a default value in the template.

Reply:
Cool, will do that